bitcoin_executor/sources/ripemd160.move

```diff
@@ -469,9 +449,35 @@ fun readLE32(v: &vector<u8>, start_index: u64): u32 {
 
 #[test]
 fun ripemd160_test() {
-    let data: vector<u8> = std::hash::sha2_256(x"0102");
-    let mut h = new();
-    h.write(data, data.length());
-    let hash = h.finalize();
-    sui::test_utils::assert_eq(hash, x"15cc49e191cbc520d91944600a5cb77af6aa3291");
+    let data = vector[
+        b"",
+        b"a",
+        b"abc",
+        b"message digest",
+        b"secure hash algorithm",
+        b"RIPEMD160 is considered to be safe",
+        b"abcdbcdecdefdefgefghfghighijhijkijkljklmklmnlmnomnopnopq",
+        b"For this sample, this 63-byte string will be used as input data",
+        b"This is exactly 64 bytes long, not counting the terminating byte",
```

Comment:
What's the behavior for larger messages?

Reply:
There was a problem with big messages, but it should work with messages over 64 bytes